reference/constraints/Locale.rst

```diff
@@ -90,5 +96,18 @@ This message is shown if the string is not a valid locale.
 
 .. include:: /reference/constraints/_payload-option.rst.inc
 
+canonicalize
+~~~~~~~~~~~~
+
+**type**: ``boolean`` **default**: ``false``
+
+.. versionadded:: 4.1
+    Using this option with value `false` was deprecated in Symfony 4.1 and it will
+    throw an exception in Symfony 5.0, use `true` instead.
+
+If true, then the :phpmethod:`Locale::canonicalize` method will be applied to
+before checking the validity of the given locale.
```

Comment:
Let's add a simple example at the end of this phrase for those who don't fully understand what "canonicalize" means or does. Example:

```rst
... before checking the validity of the given locale (e.g. ``FR-fr.utf8``
is transformed into ``fr_FR``).
```